src/components/TaskDetailsCard/index.jsx

```diff
@@ -143,7 +144,9 @@ export default class TaskDetailsCard extends Component {
                   primary="Created"
                   secondary={<DateDistance from={task.created} />}
                 />
-                <ContentCopyIcon />
+                <CopyToClipboard text={task.created}>
+                  <ContentCopyIcon />
+                </CopyToClipboard>
```

Comment:
We want the copy action to happen not only when we click the copy icon but also when we click anywhere in the list item. That said, I think we can do:

```jsx
<CopyToClipboard text={task.created}>
  <ListItem button className={classes.listItemButton}>
    <ListItemText
      primary="Created"
      secondary={<DateDistance from={task.created} />}
    />
    <ContentCopyIcon />
  </ListItem>
</CopyToClipboard>
```